themes/default/content/docs/pulumi-cloud/oidc/client/kubernetes-gke.md

```diff
@@ -0,0 +1,133 @@
+---
+title_tag: Configure OpenID Connect for Google Kubernetes Engine | OIDC
+meta_desc: This page describes how to configure Pulumi to accept Google Kubernetes Engine OIDC tokens
+title: Google Kubernetes Engine
+h1: Configuring OpenID Connect for Google Kubernetes Engine
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    pulumicloud:
+        parent: openid-connect-client
+        weight: 1
+---
+
+This document outlines the steps required to configure Pulumi to accept Google Kubernetes Engine id_tokens to be exchanged by Organization access tokens
+
+## Prerequisites
+
+* You must be an admin of your Pulumi organization.
+
+{{< notes type="warning" >}}
+Please note that this guide provides step-by-step instructions based on the official provider documentation which is subject to change. For the most current and precise information, always refer to the [official Bound Service Account Tokens documentation](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/1205-bound-service-account-tokens).
+{{< /notes >}}
+
+## Register the OIDC issuer
+
+1. Navigate to **OIDC Issuers** under your Organization's **Settings** and click on **Register a new issuer**.
+1. Name the issuer and complete the url: `https://container.googleapis.com/v1/projects/PROJECT_NAME/locations/us-west1/clusters/CLUSTER_NAME/`
+   ![Register GKE](../register-gke.png)
+1. Submit the form
+
+## Conrigure the Authorization Policies
+
+1. Click on the issuer name
+1. Change the policy decision to `Allow`
+1. Change the token type to `Organization`
+1. Add a new rule and configure it to verify the pod requesting the token: `"kubernetes.io".pod.name: runner-*`.
```

Comment:
Let's be sure to always include a namespace check in the rule(s).

But let's focus on granting based on the Kubernetes service account, not on the pod name.  Because that's closer to the intention of the feature; the "service account" is the "user" here.

Reply:
done